tests/stubtest_whitelists/py35.txt

```diff
@@ -17,6 +17,7 @@ asyncio.tasks.Task._wakeup
 asyncio.unix_events._UnixSelectorEventLoop.create_unix_server
 bdb.GENERATOR_AND_COROUTINE_FLAGS
 builtins.str.maketrans
+bz2.BZ2File.read1
```

Comment:
Why did this need to be added?

Reply:
The stub inherits this from BufferedIOBase whose definition was changed.
My other comment has the details, but I don't think that change to BufferedIOBase should be made, in which case this whitelisting will no longer be necessary.